src/ray/util/util.h

```diff
@@ -117,6 +117,38 @@ inline int64_t current_sys_time_us() {
   return mu_since_epoch.count();
 }
 
+static std::random_device              rd;
+static std::mt19937                    gen(rd());
+static std::uniform_int_distribution<> dis(0, 15);
+static std::uniform_int_distribution<> dis2(8, 11);
+
+inline std::string generate_uuid_v4() {
```

Comment:
According to our code style, it should be:
```suggestion
inline std::string GenerateUUIDV4() {
```

Reply:
According to C++ practices, we should avoid global static variables as possible. I think there are 2 options to make the code graceful:
1. Add a UUIDV4Generator class and then make `random_device` `mt19937`  as class static variables.
2. make them as local static variable, like:
```C++
inline std::string generate_uuid_v4() {
    static std::random_device              rd;
    static std::mt19937                    gen(rd());
    static std::uniform_int_distribution<> dis(0, 15);
    static std::uniform_int_distribution<> dis2(8, 11);

    std::stringstream ss;
      int i;
      ss << std::hex;
      for (i = 0; i < 8; i++) {
        ss << dis(gen);
      }
    }
```

Reply:
Yeah, I'd prefer the second one. What do you think?

Reply:
Yes. the 1st one is a bit heavy.